CMakeLists.txt

```
@@ -9,3 +9,4 @@ project(safeside VERSION 0.1.0 LANGUAGES C CXX)
 
 add_subdirectory(demos)
 add_subdirectory(kernel_modules)
+add_subdirectory(benchmarks/spectre_v1)
```

Prefer to have each build file only point one level deeper:

```suggestion
add_subdirectory(benchmarks)
```
... and then have a `CMakeLists.txt` file in `benchmarks/` with:

```cmake
add_subdirectory(spectre_v1)
```